backend/app/services/rag/runtime_resolver.py

```
@@ -90,5 +117,264 @@ def build_query_runtime_spec(
+        user_id: int,
+        model_name: str,
+        model_namespace: str,
+    ):
+        if model_namespace == "default":
+            return (
+                db.query(Kind)
+                .filter(
+                    Kind.kind == "Model",
+                    Kind.name == model_name,
+                    Kind.namespace == model_namespace,
+                    Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)